apps/3d_rec_framework/include/pcl/apps/3d_rec_framework/pipeline/impl/global_nn_recognizer_crh.hpp

```diff
@@ -48,23 +47,21 @@ pcl::rec_3d_framework::GlobalNNCRHRecognizer<Distance, PointInT, FeatureT>::getC
 {
 
   hist.reset(new CRHPointCloud);
-  std::stringstream dir;
   std::string path = source_->getModelDescriptorDir(model, training_dir_, descr_name_);
-  dir << path << "/crh_" << view_id << "_" << d_id << ".pcd";
+  std::string dir = path + "/centroid_" + std::to_string(view_id) + '_' + std::to_string(d_id) + ".pcd";
 
-  pcl::io::loadPCDFile(dir.str(), *hist);
+  pcl::io::loadPCDFile(dir, *hist);
```

Comment:
Seems this could be even simpler also:
```
std::string path = source_->getModelDescriptorDir(model, training_dir_, descr_name_) + "/centroid_" + std::to_string(view_id) + '_' + std::to_string(d_id) + ".pcd";
pcl::io::loadPCDFile(dir, *hist);
```

Reply:
Indeed this line could be combined. It is just for readability, that this is splitted - otherwise clang-format would split the line.